packages/block-editor/src/components/block-draggable/index.native.js

```diff
@@ -134,8 +148,12 @@ const BlockDraggableWrapper = ( { children } ) => {
 	};
 
 	const onChipLayout = ( { nativeEvent: { layout } } ) => {
-		chip.width.value = layout.width;
-		chip.height.value = layout.height;
+		if ( layout.width > 0 ) {
+			chip.width.value = layout.width;
+		}
+		if ( layout.height > 0 ) {
+			chip.height.value = layout.height;
+		}
```

Comment:
When hiding the chip component, we might receive a layout object with empty width and height. For this reason, we omit those values.